### ISSUE TITLE
shader bg?

### ISSUE THREAD

**Issue by kasparsj** [toplap-ka/toplap-ka-site#3]
Hellooo!
Sometimes when I do websites, I add a shader to the bg, I think this breathes some life to the statistic of the screen.
Anybody knows a good shaders (like a reference from shadertoy) we could try porting to webgl for our site background? 
Can also be very subtle/simple, like maybe just some sobtle animation with the current background colours? Wdyt?

**Comment by cappelnord** [comment]
good idea, I was also thinking about something slightly moving could be nice! As the current background is inspired by Jia's flyer graphics we could also just change site backgrounds according to a current event or something. It's made with CSS so technically this could also be just animated.

What does @luuiii think?

**Comment by luuiii** [comment]
I honestly prefer static website which could be calmly stared at, but I do had good experience with good designed moving thingies...so why not give it a try!
Maybe something like a slightly angular movement of the gradient could be nice?
